frontend/src/pages/home/index.js

```diff
@@ -72,12 +72,14 @@ function closeUsersMenu() {
 }
 
 // debounce function
+let uniqueSet = new Set();
```

Comment:
`Gotcha!!` It might be the best solution for this scenario. Good Job **`sayar win`**.